### PR TITLE
docs: add Olive release pages

### PR DESCRIPTION
**Pull request by mtyaka**
This kicks off the Olive docs as outlined on [the release process page](https://openedx.atlassian.net/wiki/spaces/COMM/pages/19662426/Process+to+Create+an+Open+edX+Release#Make-release-specific-changes-on-the-release-branch).

### Date Needed (optional)

If the release date of a feature is known or estimated, provide it to give reviewers guidance on turnaround time.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:.

- [ ] Subject matter expert:
- [ ] Subject matter expert:
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] Product review:
- [ ] Partner support:
- [ ] PM review:

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Sandbox (optional)

- [ ] Point to or build a sandbox for the software change and add a link here

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @mtyaka!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by mtyaka** [comment]
@pdpinch @nedbat This adds stub pages for the Olive release.

File: en_us/install_operations/source/platform_releases/olive.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+===============================
+Upgrading a Docker Installation
+===============================
+
+Devstack is installed using Docker. To upgrade from one Olive
+release to another, follow the instructions in `devstack`_.
+
+===============================
+Upgrading a Native Installation
```
Comment (nedbat) [inline en_us/install_operations/source/platform_releases/olive.rst]: We don't support the native installation anymore, so this section shouldn't be here, right?

**Review by nedbat — CHANGES_REQUESTED** [review]

File: en_us/install_operations/source/platform_releases/olive.rst (hunk 1 above)
Comment (mtyaka) [reply en_us/install_operations/source/platform_releases/olive.rst]: You're right, I missed that. I removed the section now.

**Comment by mtyaka** [comment]
@nedbat Do the changes look good to you now? If they do, can you please help me merge?

**Review by nedbat — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@mtyaka 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/edx-documentation&entry.867055334=https://github.com/openedx/edx-documentation/pull/2087&entry.1484655318=https://github.com/mtyaka&entry.752974735=2022-10-12+08:35&entry.1917517419=2022-10-25+17:23&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by nedbat** [comment]
@kdmccormick I did these merges (there have been three for the docs), but the process should be updated so that I don't have to be a bottleneck.

**Comment by kdmccormick** [comment]
@nedbat , would [Sarina's proposal](https://discuss.openedx.org/t/potential-new-cc-role-access-named-release-manager/8588) resolve the bottleneck? Looking at the [list of repos granted by the release manager team](https://github.com/orgs/openedx/teams/community-release-managers/repositories), I see edx-documentation, edx-developer-docs, and docs.openedx.org are all included.

**Comment by nedbat** [comment]
> @nedbat , would [Sarina's proposal](https://discuss.openedx.org/t/potential-new-cc-role-access-named-release-manager/8588) resolve the bottleneck? Looking at the [list of repos granted by the release manager team](https://github.com/orgs/openedx/teams/community-release-managers/repositories), I see edx-documentation, edx-developer-docs, and docs.openedx.org are all included.

I think it would remove me as a bottleneck, yes.  Also, if it didn't we should amend the proposal so it did!

**Pull request merged**